### PR TITLE
Ensure pre-loading for the right srcset parameter for img nodes.

### PR DESCRIPTION
**Pull request by ambroisemaupate**
Just creating the Image JS object with `srcset` parameter will load the chosen image for the given srcset rules. #64 

I updated unit tests to reflect usage of the `srcset` on `img` node.

### CODE REVIEW

File: src/jquery.waitforimages.js [hunk 1]
```diff
@@ -108,7 +108,8 @@
 
                     // If an `img` element, add it. But keep iterating in
                     // case it has a background image too.
-                    if (element.is('img:has-src')) {
+                    if (element.is('img:has-src') &&
+                        !element[0].hasAttribute('srcset')) {
```
Comment (alexanderdickson) [inline src/jquery.waitforimages.js]: How about `element.has('[srcset]')`?

File: src/jquery.waitforimages.js [hunk 2]
```diff
@@ -209,6 +203,9 @@
 
                 });
 
+                if (typeof img.srcset !== "undefined") {
```
Comment (alexanderdickson) [inline src/jquery.waitforimages.js]: no need for `typeof`, just check `img.srcset`

File: src/jquery.waitforimages.js (hunk 1 above)
Comment (ambroisemaupate) [inline src/jquery.waitforimages.js]: @alexanderdickson I don’t understand your advice. `[srcset]` is already part of `hasImageAttributes`.

File: src/jquery.waitforimages.js (hunk 2 above)
Comment (ambroisemaupate) [inline src/jquery.waitforimages.js]: It’s safer to check against `"undefined"` for browsers that do not support `srcset`.

File: src/jquery.waitforimages.js (hunk 1 above)
Comment (alexanderdickson) [inline src/jquery.waitforimages.js]: I mean the long-winded `element[0].hasAttribute('srcset')` could be replaced with `element.is('[srcset]')`

File: src/jquery.waitforimages.js (hunk 2 above)
Comment (alexanderdickson) [inline src/jquery.waitforimages.js]: Why is it safer?

If the property doesnt exist, it will be `undefined`, which is falsey.

**Comment by ambroisemaupate** [comment]
Hi @alexanderdickson do you have any more suggestions regarding this pull request?
Tnanks for your time.

**Comment by alexanderdickson** [comment]
Thanks @ambroisemaupate

**Pull request merged**